freqtrade/exchange/__init__.py

```diff
@@ -1,3 +1,8 @@
 from freqtrade.exchange.exchange import Exchange  # noqa: F401
+from freqtrade.exchange.exchange import (is_exchange_supported,  # noqa: F401
+                                         supported_exchanges)
+from freqtrade.exchange.exchange import (timeframe_to_seconds,  # noqa: F401
```

Comment:
Why don't you combine these 2 imports ... ? they are multiline anyway, so you add them within the same set of parenteses...

Reply:
Errhm, "for historical reasons". ;-) They (those two sets of functions) came in two different commits, from two different reasons...